packages/core/test/tabs/tabsTests.tsx

```diff
@@ -128,6 +129,21 @@ describe("<Tabs>", () => {
         assert.lengthOf(wrapper.find(`.${panelClassName}`), 1);
     });
 
+    it("passes correct tabTitleId and tabPanelId to panel renderer", () => {
+        mount(
+            <Tabs id={ID}>
+                <Tab
+                    id="first"
+                    panel={({ tabTitleId, tabPanelId }) => {
+                        assert.equal(tabTitleId, generateTabTitleId(ID, "first"));
+                        assert.equal(tabPanelId, generateTabPanelId(ID, "first"));
```

Comment:
are you sure these assertions are getting run in this test? I think you need to use the Mocha async `done` syntax.

Reply:
Yeah I was a little confused on how to write this test, was curious how you would do it.

> are you sure these assertions are getting run in this test?

Yep, it does work. If I change the expected value, I see a failure.